src/iam/provisionmanager/provisionmanager.cpp

```diff
@@ -0,0 +1,118 @@
+#include "aos/iam/provisionmanager.hpp"
+#include "log.hpp"
+
+namespace aos::iam::provisionmanager {
+
+/***********************************************************************************************************************
+ * Public
+ **********************************************************************************************************************/
+
+Error ProvisionManager::Init(const aos::Array<CertTypeConfig>& certTypeConfigs, ProvisionManagerCallback& callback,
+    certhandler::CertHandlerItf& certHandler)
+{
+    LOG_DBG() << "Init provision manager";
+
+    mCallback        = &callback;
+    mCertHandler     = &certHandler;
+    mCertTypeConfigs = certTypeConfigs;
+
+    return aos::ErrorEnum::eNone;
+}
+
+Error ProvisionManager::StartProvisioning(const String& password)
+{
+    LOG_DBG() << "Start provisioning";
+
+    auto err = mCallback->OnStartProvisioning();
+    if (!err.IsNone()) {
+        return err;
+    }
+
+    for (const auto& certTypeConfig : mCertTypeConfigs) {
+        LOG_DBG() << "Clearing cert type" << certTypeConfig.mCertType;
```

Comment:
Use present simple in log. And format it as following:
`Clear cert storage: type = ...`